src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisController.java

```diff
@@ -97,6 +98,16 @@ public ResponseEntity<UUID> runAndSave(@Parameter(description = "Network UUID")
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(resultUuid);
     }
 
+    @PostMapping(value = "/networks/{networkUuid}/computation-count", produces = APPLICATION_JSON_VALUE)
```

Comment:
/networks/{networkUuid}/parameters/complexity ?

Reply:
It's a proposal from Slimane